lib/go-tc/profile_parameters.go

```diff
@@ -59,3 +61,50 @@ type ProfileParameterNullable struct {
 	Parameter   *string    `json:"parameter" db:"parameter"`
 	ParameterID *int       `json:"parameterId" db:"parameter_id"`
 }
+
+// ProfileParametersResponseV5 is the type of the response from Traffic Ops to
+// GET requests made to its /profileparameters API endpoint.
+type ProfileParametersResponseV5 struct {
+	Response []ProfileParameterV5 `json:"response"`
+	Alerts
+}
+
+// ProfileParameterResponseV5 is a single ProfileParameter response for Create to
+// depict what changed.
+// swagger:response ProfileParameterResponse
+// in: body
+type ProfileParameterResponseV5 struct {
+	// in: body
+	Response ProfileParameterV5 `json:"response"`
+	Alerts
+}
+
+// ProfileParameterV5 is the latest minor version of the major version 5
+type ProfileParameterV5 ProfileParameterV50
+
+// ProfileParameterV50 is a representation of a relationship between a Parameter
+// and a Profile to which it is assigned.
+//
+// Note that not all unique identifiers for each represented object in this
+// relationship structure are guaranteed to be populated by the Traffic Ops
+// API.
+type ProfileParameterV50 struct {
+	LastUpdated time.Time `json:"lastUpdated"`
+	Profile     string    `json:"profile"`
+	ProfileID   int       `json:"profileId"`
+	Parameter   string    `json:"parameter"`
+	ParameterID int       `json:"parameterId"`
+}
+
+// ProfileParameterNullableV5 is the latest minor version of the major version 5
```

Comment:
You don't seem to using this struct anywhere in the codebase, then why define it?

Reply:
Fixed It.